quartz/src/main/java/org/quartz/core/jmx/QuartzSchedulerMBean.java

```diff
@@ -183,7 +183,7 @@ boolean deleteJob(String jobName, String jobGroupName)
      * Maps are limited to containing values from the same set of types, less Map itself.
      * 
      * @param abstractJobInfo map of attributes defining job
-     * @param replace whether or not to replace a pre-existing job with the same key
+     * @param replace whether or not to replace a preexisting job with the same key
```

Comment:
I think both pre-existing and preexisting are correct, but I guess not a major deal. (https://dictionary.cambridge.org/dictionary/english/pre-existing)